community/cypher/cypher/src/test/scala/org/neo4j/cypher/CartesianProductNotificationAcceptanceTest.scala

```diff
@@ -70,10 +69,10 @@ class CartesianProductNotificationAcceptanceTest extends CypherFunSuite with Gra
     }
 
     //then
-    verify(logger, times(1)) += CartesianProductNotification(InputPosition(0, 1, 1), Set("x", "y"))
+    verify(logger) += CartesianProductNotification(Set("x", "y"))
   }
 
-  test("should not warn when disconnected patterns in multiple match clauses") {
+  test("should warn when disconnected patterns in multiple match clauses") {
```

Comment:
[My comment disappeared] Why have we changed our minds on this? The original intention as stated in the CIP is that we should not warn when the user is explicit about wanting a CartesianProduct.